src/openforms/utils/templatetags/utils.py

```diff
@@ -0,0 +1,8 @@
+from django import template
```

Comment:
Probably wise to put a big fat disclaimer at the top that any tags and filters defined here are automatically enabled for user-submitted templates.